systemd/nightly-stathat-success.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+curl -d "stat=tuxbot - nightly - success&email=$STATHAT_EZKEY&value=1" http://api.stathat.com/ez
```

Comment:
preferred parameter is `ezkey`.  They both work, sorry if docs say something else.

And use `https://api.stathat.com/ez` instead of `http`.